geosys/bridge_api/test/test_field_level_maps.py

```diff
@@ -0,0 +1,65 @@
+# coding=utf-8
+"""Bridge API connection test.
+
+.. note:: This program is free software; you can redistribute it and/or modify
+     it under the terms of the GNU General Public License as published by
+     the Free Software Foundation; either version 2 of the License, or
+     (at your option) any later version.
+
+"""
+import unittest
+
+from geosys.bridge_api.connection import ConnectionAPIClientV2
```

Comment:
I don't know that we should be baking in the API version into our symbol names? Or do you think it is better? Perhaps we can parameterise these as class members rather in the relevant classes?

Reply:
Just in case different API version has a significantly different schema of API. I was trying to avoid `if else` conditional implementation if that case happens. Or do you think it's still better to do the conditional implementation for each API version?